### PR TITLE
(release) Release 3.0.0

### PR DESCRIPTION
**Pull request by denniskigen**
Releases v3.0.0. Read more about the rationale [here](https://talk.openmrs.org/t/o3-and-3-0-0-its-time-to-say-goodbye-to-the-beta/43054).

### CODE REVIEW

File: distro/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.openmrs.distro</groupId>
     <artifactId>referenceapplication</artifactId>
-    <version>3.0.0</version>
+    <version>3.0.1-SNAPSHOT</version>
```
Comment (ibacher) [inline distro/pom.xml:7]: Small remark: I'd set this to 3.1.0-SNAPSHOT.

**Comment by jayasanka-sack** [comment]
We need to release the core to include this bug fix: https://github.com/openmrs/openmrs-core/commit/dec07a0c4771d8344608f98c5bcc6b0331930e63

**Comment by jayasanka-sack** [comment]
The build is failing because it can't find the initializer 2.7.0. @mks-d any idea?

```
Error:  Failed to execute goal on project distro-emr-configuration: Could not resolve dependencies for project org.openmrs:distro-emr-configuration:pom:3.1.0-SNAPSHOT: Could not find artifact org.openmrs.module:initializer-omod:jar:2.7.0 in openmrs-repo (https://mavenrepo.openmrs.org/public) -> [Help 1]
```

**Comment by ibacher** [comment]
The glitch with the 2.7.0 release has been fixed, so re-running these.

**Review by ibacher — APPROVED** [review]

**Pull request merged**